src/ducks/demo/AugmentedModal.styl

```diff
@@ -1,32 +1,35 @@
-.AugmentedModalIntent
-    padding 0
+.ContainerIntentSide
+    display flex
+    height 100%
+    padding 0 !important
 
-.AugmentedModalDescription--NoHeader
-    margin-top 0 !important
-
-.AugmentedModalDescription
-    padding 0!important
-    & > div
-        height 100%
-        & > *
-            box-sizing border-box
-
-            &:last-child
-                flex 0 1 35%
-                padding 1.5rem
+.IntentIframeContainer
+    flex 5
 
 .FakeInfos
     position relative
+    flex 2
+    padding 20px
+    background-color #F5F6F7
+    overflow auto
 
 .FakeInfosLinkVendor
 .FakeInfosLinkDrive
     position absolute
-    left 0
-    width 125px
+    left 20px
     height 25px
 
 .FakeInfosLinkVendor
-    top 203px
+    width 140px
+    top 225px
 
 .FakeInfosLinkDrive
-    top 359px
+    width 104px
+    top 380px
+
+.FakeVeePeeHeader
+    background-color #eee
```

Comment:
Default background color could (--defaultBackgroundColor) be used here.

Reply:
Thx I changed the line 13 which contained #F5F6F7 (defaultBackgroundColor), on the other hand i didn't find semantic value about #eee (L31).

Reply:
OK, it's weird that we use it then, by any chance, do you know why ? Ideally, when doing a migration it's good to also check with the design/product to see if we can normalize/put in coherence the visuals so that we do not have to do adhoc styling.

Anyway, I understand that this is only for demos, maybe the adhocness is justified. Good for me.

Reply:
There was a discussion with Ben and he wanted to keep this display. :)